### ISSUE TITLE
[Question] Is there a way to see what's in the queue?

### ISSUE THREAD

**Issue by ojmaster** [mylar3/mylar3#1352]
Like a list of whats in queue or what tasks are running, rather than having to wait for things to pop up in console

**Comment by evilhero** [comment]
No, there is no ability to monitor any of the queues that mylar utilizes. No other application has the ability to show what's happening in the queues they use, and mylar is no different.

That being said, there was some queue display aspects that were being worked on but haven't been touched in a while due to other commitments.
